### PR TITLE
Record start/end time & duration of test checks

### PR DESCRIPTION
**Pull request by happz**

### CODE REVIEW

**Review by therazix — APPROVED** [review]

**Review by psss — CHANGES_REQUESTED** [review]
Looks nice! Just two little nitpicks...

**Comment by psss** [comment]
Rebased and resolved the conflict.

File: tmt/steps/execute/internal.py [hunk 1]
```diff
@@ -435,10 +430,23 @@ def _run_tests(
...
+                logger.verbose(
+                    f"{_format_duration(result)} {result.show()} [{progress}]",
+                    shift=shift)
+
+                for check_result in result.check:
+                    logger.verbose(
+                        f'{_format_duration(check_result)} '
+                        f'{check_result.show()} '
+                        f'({check_result.event.value} check)',
+                        shift=shift)
```
[the remark this answers is not in the document]
Comment (happz) [reply tmt/steps/execute/internal.py:448]: I went with this:

```
        exit-first: false
            test: /avc/harmless
                cmd: /bin/true
                00:00:00 pass /avc/harmless (on default-0) [1/1]
                00:00:00     pass avc (after-test check)
```

The "<result> <name>" block is handled by `Result.show()`, it's harder to introduce indentation into that. I want to change this but with another patch, I'm not comfortable with `Result` formatting stuff for others, and it also ignores colorization controls.

[the remark this answers is not in the document]
Comment (psss) [reply tmt/steps/execute/internal.py:453]: Yeah, makes sense.

**Comment by happz** [comment]
Rebased and resolved the conflict.

**Review by psss — APPROVED** [review]
Thanks!

**Pull request merged**